cpt/src/main/java/org/isisaddons/wicket/wickedcharts/cpt/applib/Base64Serializer.java

```diff
@@ -0,0 +1,63 @@
+package org.isisaddons.wicket.wickedcharts.cpt.applib;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+import org.apache.commons.codec.binary.Base64;
+
+
+public class Base64Serializer {
```

Comment:
Is this class really needed ?
Java 8 provides `java.util.Base64`.
Wicket also provides `org.apache.wicket.util.crypt.Base64` (deprecated in Wicket 8.x)